### ISSUE TITLE
Is it possible to use this for WatchOS

### ISSUE THREAD

**Issue by auro-krishna** [orta/cocoapods-keys#207]

**Comment by ashfurrow** [comment]
I'm not sure, I've never built a WatchOS app. If anyone else knows, please sound off 🗣️

**Comment by SlaunchaMan** [comment]
Yes, it is definitely possible—it even works in a SwiftUI preview!

<img width="2672" alt="Screen Shot 2020-07-13 at 9 53 34 AM" src="https://user-images.githubusercontent.com/147458/87312619-cd122380-c4ee-11ea-92ea-9921e4b1023a.png">

**Comment by quver** [comment]
@SlaunchaMan could you show your PodFile?

**Comment by SlaunchaMan** [comment]
Sorry, I deleted this sample project, but I don’t think it was any different than a regular iOS app using CocoaPods-Keys.
